reactDashboard/src/pages/Regional.jsx

```diff
@@ -129,16 +146,15 @@ const Regional = ({
         </Col>
       </Row>
       <Row className="fullscreen-charts">
-        <Col xs={12} md={3} lg={2}>
-          <RegionDataChartsSelector
-            chartType={chartType}
-            setChartType={setChartType}
-          />
-        </Col>
-        <Col xs={12} md={9} lg={10}>
+        <Col xs={12}>
           <RegionDataCharts
             chartType={chartType}
+            setChartType={setChartType}
```

Comment:
The only reason chartType exists outside the new DataCharts/RegionalDataCharts is to allow stopping of any playing sonification when the user selects a different chart. It may be cleaner to move the stopAudio() effect (line 106) into the dataChart components along with the chartType/setChartType state. That way it's more self contained within the charts, and the regional and overview pages don't need to care about chartType state.

Reply:
moved to separate ticket